src/tokens.scss

```
@@ -0,0 +1,5 @@
+:host {
+  --ds-auro-drawer-background-color: var(--ds-color-background-primary-100-default, $ds-color-background-primary-100-default);
+  --ds-auro-drawer-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+  --ds-auro-drawer-icon-color: var(--ds-color-icon-primary-default, $ds-color-icon-primary-default);
```

I think this token needs to be more specifically named. It reads like its for any icon anywhere in the drawer much like `--ds-auro-drawer-text-color` affects all text in the drawer.

```suggestion
  --ds-auro-drawer-close-button-color: var(--ds-color-icon-primary-default, $ds-color-icon-primary-default);
```

Or maybe `--ds-auro-drawer-close-button-icon-color` if you prefer.

---

Another thing - This should have been an `auro-button` so that it was a11y accessible. Since we are creating theme tokens based on the content it may be the right time to wrap that in an auro-button and style it using that compoents Tier 3 token structure.

---

With so many things moving at once (15-20 token PRs), I feel like it would be better to create a ticket for it. If you feel strongly about this then lets discuss